roles/hello-world-apb/tasks/verify_deprovision.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: "Verify {{ app_name }} objects removed"
```

Comment:
@fabianvf @dymurray I scrapped the use of lookups for verification because they don't work as you would expect and I think this is cleaner and makes more sense. I'll consider writing an issue against Ansible, but for now, I think this is the most correct way to do this.